students/Zach Cooper/lesson03/test_basic_operations.py

```diff
@@ -0,0 +1,138 @@
+import pytest
+import customers_model as cm
+import basic_operations as bo
+
+ok_customer1 = {'customer_id': 'W3434fd',
+                'first_name': 'Suzie',
+                'last_name': 'Edgar',
+                'home_address': '123 Browncroft Blvd, Rochester, NY 97235',
+                'phone_number': '234-123-4567',
+                'email_address': 'suzie@hotmail.com',
+                'status': True,
+                'credit_limit': 40}
+
+
+ok_customer2 = {'customer_id': '2',
+                'first_name': 'Chris',
+                'last_name': 'Seefeld',
+                'home_address': '301 Boston Commons Way, Salem, 98028',
+                'phone_number': '123-456-4566',
+                'email_address': 'Chris@hotmail.com',
+                'status': True,
+                'credit_limit': 200}
+
+
+ok_customer3 = {'customer_id': '3',
+                'first_name': 'Ryan',
+                'last_name': 'Fermstad',
+                'home_address': '808 Honolulu Way, Dimond Head, 80808',
+                'phone_number': '765-432-1234',
+                'email_address': 'fermy@aol.com',
+                'status': False,
+                'credit_limit': 300.00}
+
+
+def clear_database():
+    cm.database.drop_tables([
+        cm.Customer
+    ])
+    cm.database.close()
+
+
+def create_empty_database():
+    clear_database()
+    cm.database.create_tables([
+        cm.Customer
+    ])
+    cm.database.close()
```

Comment:
You don't have to close the connection every test. This only needs to be done at the very end.